src/EasyCaching.Redis/DefaultRedisCachingProvider.cs

```diff
@@ -343,6 +349,9 @@ private string HandlePrefix(string prefix)
             if (!prefix.EndsWith("*", StringComparison.OrdinalIgnoreCase))
                 prefix = string.Concat(prefix, "*");
 
+            if (!string.IsNullOrWhiteSpace(_options.DBConfig.KeyPrefix))
```

Comment:
Users may use **_options.DBConfig.Configuration** for **KeyPrefix**.

I think it will be better to handle this one.

Reply:
Can I ask why there is multiple ways to set a keyprefix? :-)

Reply:
We actually use _options.DBConfig.KeyPrefix is in our project. I assume we should handle both?

Reply:
looking into docs, it does not seem like the KeyPrefix can be set through the configuration string, see:
https://stackexchange.github.io/StackExchange.Redis/Configuration.html

Reply:
I am so sorry!!  It's only available in csredis.